GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -45,6 +44,9 @@ public sealed partial class FormCommit : GitModuleForm
 
         #region Translation
 
+        private readonly TranslationString _commitAndPush = new("Commit && &push");
```

Comment:
Is there a stray translation linger somewhere?

Reply:
Do you mean the value from the designer?
https://github.com/gitextensions/gitextensions/blob/63ba58e3579381edc1d374774ba1233bdce4d9d9/GitUI/CommandsDialogs/FormCommit.Designer.cs#L1150
How can these translation strings be merged / reused so that the text is still visible in the designer?

Reply:
Remove in designer